README.md

```diff
@@ -60,6 +60,15 @@ export PATH=$PATH:$HOME/.local/bin
 poetry install
 ```
 
+### PlantUML example
+Files of a single type can be "translated" to PlantUML diagrams. See the following example:
+
+```shell
+janus translate --input-dir INPUT_DIR --output-dir OUTPUT_DIR --source-lang SOURCE_LANG --target-lang plantuml --prompt-template janus-llm/janus/prompts/templates/uml-diagram/
+```
+
+Please note that using plantuml as an input language in regular translation is very likely broken.
```

Comment:
Just curious, why would using it as an inputdir be broken? Is it because translating from UML to a language like Python doesn't make much sense?

Reply:
Yes, not only does it not make much sense but because of the tree-sitter problem described earlier it just wouldn't function that way at the moment.